package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Render Gavel.js output to HTML diff",
   "main": "lib/index.js",
   "scripts": {
-    "build": "coffee -b -c -o lib/ src/",
+    "build": "npm run build:coffee-script && npm run build:html-sanitizer",
+    "build:coffee-script": "coffee -b -c -o lib/ src/",
+    "build:html-sanitizer": "node -e 'fs = require(\"fs\"); fs.createReadStream(\"src/html-sanitizer.js\").pipe(fs.createWriteStream(\"lib/html-sanitizer.js\"))'",
```

Comment:
This is one-liner which manually copies the `html-sanitizer.js` file. It's JavaScript, because we need it to be portable (*nix, windows).